projects/core/src/datetimepicker/calendar.scss

```diff
@@ -1,173 +1,162 @@
 @import '~@angular/material/theming';
-
 $mat-calendar-padding: 8px !default;
 $mat-calendar-weekday-table-font-size: 11px !default;
-
 // Values chosen to approximate https://material.io/icons/#ic_navigate_before and
 // https://material.io/icons/#ic_navigate_next as closely as possible.
 $mat-calendar-prev-next-icon-border-width: 2px;
 $mat-calendar-prev-next-icon-margin: 15.5px;
 $mat-calendar-prev-icon-transform: translateX(2px) rotate(-45deg);
 $mat-calendar-next-icon-transform: translateX(-2px) rotate(45deg);
-
 .mat-datetimepicker-calendar {
-  @include user-select(none);
-  display: block;
-  outline: none;
-
-  &[mode='landscape'] {
-    display: flex;
-  }
+    @include user-select(none);
```

Comment:
please stick to two space indentation. I cannot see your changes this way!

Reply:
Done